src/components/LazyLoader/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import Skeleton, { SkeletonTheme } from 'react-loading-skeleton'
+
+const LazyLoader = (): JSX.Element => {
+  const themeColor = '#313236'
+  const highlightColor = '#49494D'
+
+  return (
+    <div className="container-lazy-loader" data-test-id="lazyLoader">
```

Comment:
I think just `lazy-loader` here is just fine. No need to define it as container. 💭

Reply:
Fixed in dcd718357360229c83952455bdf0758a7380e5df